storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -953,6 +954,20 @@ void Archived_Redo_Log_Monitor::thread_func() {
     std::string dir;
     std::string subdir;
   } archive;
+  std::ostringstream query;
+  read_mysql_variables(mysql, "SHOW VARIABLES", vars, true);
```

Comment:
is this necessary, having in mind we do the exact same query a few lines below ?

Reply:
Yes，need to check   SET GLOBAL innodb_redo_log_archive_dirs  takes effect or not